GeoFire.podspec

```diff
@@ -1,15 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "GeoFire"
-  s.version      = "2.0.0"
+  s.version      = "2.0.1"
```

Comment:
Perhaps should be 3.0.0 since presumably there are breaking changes in the dependencies moving from Database 2.x to 4.x?